config/app.php

```diff
@@ -44,5 +44,6 @@
         Toby\Architecture\Providers\RouteServiceProvider::class,
         Toby\Architecture\Providers\TelescopeServiceProvider::class,
         Toby\Architecture\Providers\ObserverServiceProvider::class,
+        Toby\Providers\DuskServiceProvider::class,
```

Comment:
I would add this provider as a optional because production image does not have dev dependencies and he should not be build properly.
We have another option, move Dusk depedency from `require-dev` to `require` too.
What do u think @mtracz?

Reply:
Agree, I would like to register this provider only in non production environments.
You can do it in AppServiceProvider, in the `register()` method, and check if environment is not `production` then register this provider here.